contracts/axelar-gas-service/tests/test.rs

```diff
@@ -28,6 +26,20 @@ fn setup_env<'a>() -> (Env, Address, Address, AxelarGasServiceClient<'a>) {
     (env, contract_id, gas_collector, client)
 }
 
+fn setup_gas_token(env: &Env, gas_amount: i128) -> Token {
+    let asset = env.register_stellar_asset_contract_v2(Address::generate(env));
+    Token {
```

Comment:
mint here to avoid needing another method

Reply:
there's some overlap with Noah's testutils PR as well btw

Reply:
for a couple tests that require specific setup, the logic can be repeated instead of using the helper